src/components/application_manager/src/commands/mobile/set_app_icon_request.cc

```diff
@@ -253,6 +249,10 @@ void SetAppIconRequest::on_event(const event_engine::Event& event) {
         const std::string& path =
             (*message_)[strings::msg_params][strings::sync_file_name]
                        [strings::value].asString();
+        if (is_icons_saving_enabled_) {
```

Comment:
@VVeremjova OK, I understand for this RPC that has only one HMI interface in this case result_code and result will have same values. I agree